apps/api/src/domain/models/video/commands/constants.ts

```diff
@@ -1 +1,3 @@
 export const CREATE_VIDEO = 'CREATE_VIDEO';
+
+export const TRANSLATE_VIDEO_NAME = 'TRANSLATE_VIDEO_NAME';
```

Comment:
Be sure to add one for `VIDEO_NAME_TRANSLATED` (the event).

Reply:
done